news/2025/2025-09-30-moscow-osu-event-2025-recap.md

```diff
@@ -0,0 +1,131 @@
+
+### osu!mania 4K
+
+The osu!mania 4K Tournament featured ::{ flag=RU }:: [-Oni-Chan-](https://osu.ppy.sh/users/10036005), ::{ flag=RU }:: [Dystire](https://osu.ppy.sh/users/11585213), ::{ flag=RU }:: [kloofhi](https://osu.ppy.sh/users/29627572) and ::{ flag=RU }:: [VikaGamer228](https://osu.ppy.sh/users/20049082). Two of these players were playing the [osu!mania 4K World Cup 2024](https://osu.ppy.sh/wiki/en/Tournaments/MWC/2024_4K)!
+### Matches summary
+| Player A |  |  | Player B | VOD link |
+| --: | :-: | :-: | :-- | :-: |
+| **[Dystire](https://osu.ppy.sh/users/11585213)** ::{ flag=RU }:: | **4** | 0 | ::{ flag=RU }:: [VikaGamer228](https://osu.ppy.sh/users/20049082) | [#1](https://youtu.be/LzItzsD4nH0?si=7_QiOYSxjuCsEP9J&t=3745) |
+| [kloofhi](https://osu.ppy.sh/users/29627572) ::{ flag=RU }:: | 2 | **4** | ::{ flag=RU }:: **[-Oni-Chan-](https://osu.ppy.sh/users/10036005)** | [#1](https://youtu.be/LzItzsD4nH0?si=wbMCtw5I7Oky3HnG&t=6429) |
+| **[Dystire](https://osu.ppy.sh/users/11585213)** ::{ flag=RU }:: | **5** | 1 | ::{ flag=RU }:: [-Oni-Chan-](https://osu.ppy.sh/users/10036005) | [#1](https://youtu.be/fIpS-LFpB7w?si=7nuxuyU9vzfVs8x2&t=2811) |
+
+### osu!mania 7K
+
+The osu!mania 7K Tournament featured ::{ flag=RU }:: [karayss](https://osu.ppy.sh/users/7156634), ::{ flag=RU }:: [PhobosX](https://osu.ppy.sh/users/2570019), ::{ flag=RU }:: [timebuka](https://osu.ppy.sh/users/6857841) and ::{ flag=RU }:: [X_Devil](https://osu.ppy.sh/users/6694908). Two of these players were playing the [osu!mania 7K World Cup 2025](https://osu.ppy.sh/wiki/en/Tournaments/MWC/2025_7K)!!
+### Matches summary
+| Player A |  |  | Player B | VOD link |
+| --: | :-: | :-: | :-- | :-: |
+| [PhobosX](https://osu.ppy.sh/users/2570019) ::{ flag=RU }:: | 0 | **4** | ::{ flag=RU }:: **[karayss](https://osu.ppy.sh/users/7156634)** | [#1](https://youtu.be/xatn-BXpMho?si=zvoQEqysGk8LdfF9&t=19017) |
+| [X_Devil](https://osu.ppy.sh/users/6694908) ::{ flag=RU }:: | 3 | **4** | ::{ flag=RU }:: **[timebuka](https://osu.ppy.sh/users/6857841)** | [#1](https://youtu.be/xatn-BXpMho?si=s6pxmtqLG7EN_N02&t=22857) |
+| [karayss](https://osu.ppy.sh/users/7156634) ::{ flag=RU }:: | 4 | **5** | ::{ flag=RU }:: **[timebuka](https://osu.ppy.sh/users/6857841)** | [#1](https://youtu.be/fIpS-LFpB7w?si=jr-1rOcViRSAGrm6&t=13209) |
+
+_Fun fact: the osu!mania 7K Tournament had more matches that went to Tiebreaker than all other gamemodes together (including standard)_
+
+---
+
+More than **450** people have visited Moscow osu! Event 2025, with **51** individual contributors and staff members making sure everything goes smooth! We are very thankful for everyone taking part in it and hope that the next iterations will be even bigger and more fun! Make sure to follow Moscow osu! Event's [Discord server](https://discord.gg/moscowosuevent), [Twitter](https://x.com/moscowosuevent), [Telegram](https://t.me/moscowosuevent) or [VK](https://vk.com/moscowosuevent) so you don't miss the next iteration!
```

Comment:
```suggestion
More than **450** people visited Moscow osu! Event 2025, with **51** individual contributors and staff members making sure everything goes smooth. We are very grateful to all participants and hope that next iterations will be even bigger and bring even more fun! Make sure to follow Moscow osu! Event's [Discord server](https://discord.gg/moscowosuevent), [Twitter](https://x.com/moscowosuevent), [Telegram](https://t.me/moscowosuevent), or [VK](https://vk.com/moscowosuevent), so that you don't miss the next iteration!
```
too many exclamations (also not sure what to replace the last "iteration" with -- maybe "gathering"?)